src/render/mod.rs

```diff
@@ -130,6 +130,7 @@ pub fn render_main_window(
                         a(href="/step/single_back") { div(title="Execute previous MIR statement/terminator (restarts and steps till one stmt before the current stmt)") { : "Step back (slow)" } }
                         a(href="/step/continue") { div(title="Run until termination or breakpoint") { : "Continue" } }
                         a(href="/step/restart") { div(title="Abort execution and restart") { : "Restart" } }
+                        a(href="/step/main") { div(title="Continue to the start of `main`") { : "-> Main" } }
```

Comment:
I did prefer if this button would be hidden once you reach main.

Reply:
Hmm, that's a good point. I mostly did this to avoid needing to read the dsl from `horrorshow`.